packages/py3_pycparser.rb

```diff
@@ -7,4 +7,5 @@ class Py3_pycparser < Pip
   license 'BSD-3'
   compatibility 'all'
   source_url 'SKIP'
+  binary_compression 'tar.zst'
```

Comment:
Why is there a `binary_compression` value in this package if it does not have any `binary_sha256` values?